lib/Transformation.ts

```diff
@@ -197,3 +197,24 @@ export function transformAggregate(expr: Alg.AggregateExpression) {
 export function transformExistence(expr: Alg.ExistenceExpression) {
   return new E.Existence(expr);
 }
+
+// Ordering 2 given strings
+export function orderTypes(litA: RDF.NamedNode | RDF.BlankNode | RDF.Literal | RDF.Variable | RDF.DefaultGraph | undefined, 
```

Comment:
`RDF.NamedNode | RDF.BlankNode | RDF.Literal | RDF.Variable | RDF.DefaultGraph` can be simplified to `RDF.Term`.

Also, I would move these two new functions into a new file (`Ordering.ts`), as this is not really a transformation.

Reply:
I'm also wondering now if literals can really be undefined. What happens if you remove these undefined-checks?

Reply:
const transform = async (bindings: Bindings, next: any) => {
        try {
          const result = await evaluator.evaluate(bindings);
          transformedStream._push({ bindings, result });
          
        } catch (err) {
          if (!isExpressionError(err)) {
            bindingsStream.emit('error', err);
          }
          transformedStream._push({ bindings, result: undefined });
        }
        next();
      };
In the ActorQueryOperationOrderBySparqlee there is a possibility of having an error. If for example I would pass a non-literal into the evaluate it will throw an error and then still do the ordering but with undefined.